schema.graphql

```diff
@@ -66,6 +66,54 @@ type Extrinsic @entity {
   specVersionId: Int!
 }
 
+type AssetPermissions @jsonField {
+  type: String!
+  values: [String]!
+}
+
+type TransactionPermissions @jsonField {
+  type: String!
+  values: [String]!
+}
+
+type PortfolioPermissions @jsonField {
+  type: String!
+  values: [PortfolioJson]!
```

Comment:
No need for a change, but some thoughts.

I think Portfolios will get their own Entity eventually, and it would be nice to have the GraphQL edge, which I don't think we can get with JSON.

I think once we get a full Portofolio Entity we might want to migrate this. Any thoughts on this?

Reply:
Makes sense. I guess a benefit of having to resync every deploy is we don't have to worry about data migration problems.

I'd be more apprehensive if this was a normal DB and we'd have to migrate over the data.